jsapp/js/components/table.es6

```diff
@@ -405,6 +404,10 @@ export class DataTable extends React.Component {
         filterable: false,
         Cell: row => {
             if (showLabels && q && q.type && row.value) {
+              if (q.type == 'image' || q.type == 'audio' || q.type == 'video') {
```

Comment:
There is a list of question types in `constants.es6`, please use the values from there instead of strings (e.g. `q.type === QUESTION_TYPES.get('audio').id`. Also please use triple equals instead of double :)

Reply:
Should I make the same changes here? https://github.com/kobotoolbox/kpi/blob/2d41d4f4f574313a4d18b8e3ff87aa45c52c230e/jsapp/js/components/table.es6#L412-L416

Reply:
I think you can leave it as is - ideally we would want all the code to use the constant, but let's try to only update the parts of code we touch :)